Objects/complexobject.c

```diff
@@ -43,6 +58,22 @@ _Py_c_diff(Py_complex a, Py_complex b)
     return r;
 }
 
+Py_complex
+_Py_cr_diff(Py_complex a, double b)
+{
+    Py_complex r = a;
+    r.real -= b;
+    return r;
+}
+
+Py_complex
+_Py_rc_diff(double a, Py_complex b)
+{
+    Py_complex r = {a, -b.imag};
+    r.real -= b.real;
+    return r;
```

Comment:
```suggestion
    Py_complex r;
    r.real = a - b.real;
    r.imag = -b.imag;
    return r;
```

Or was there any reason to use what you wrote? (why not just returning `{a - b.real, -b.imag}` also?)